docs/user_guide/intelligence_dashboard.md

```diff
@@ -108,4 +113,4 @@ This dashboard displays the following information about devices:
 
 (The screenshot above does not include the full dashboard)
 
-The Asset Intelligence dashboard has been added to the Cyences app in version 1.4.0.
\ No newline at end of file
+The "Search Users and Devices" dashboard has been added to the Cyences app in version 1.4.0.
```

Comment:
The "Search Users and Devices" dashboard has been added to the Cyences app in version 1.4.0, initially named "Asset Intellience", later renamed to "Search Users and Devices" in version 4.3.0.